i8_terminal/commands/watchlist/watchlist_metrics.py

```diff
@@ -50,6 +54,14 @@ def metrics(name: str, metrics: str, export_path: Optional[str]) -> None:
 
     """
     console = Console()
+    if not metrics and not view_name:
+        console.print("The metrics or view_name options must be entered", style="yellow")
+        return
+    if view_name and metrics:
+        console.print("The metrics and view_name options are not compatible to use together", style="yellow")
```

Comment:
Always use quotation for specific names. In this case "The 'metrics' or 'view_name' options are mutually exclusive."